pancurses pulls ncurses-rs which breaks cross (termion used instead)

diff --git a/Cargo.toml b/Cargo.toml
index 576332a..8023123 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,7 +47,6 @@ strum_macros = "0.22.0"
 regex = "1"
 ioctl-rs = { version = "0.2", optional = true }
 serde_cbor = "0.11.2"
-pancurses = { version = "0.17.0", features = ["win32"] }

 [dependencies.cursive]
 version = "0.16.3"
